### PR TITLE
Simultaneous unit test

### PR DESCRIPTION
**Pull request by tbssajal**
Added unit tests for api methods in ValidatorServiceWeb3 and BlockchainServiceWeb3.

### CODE REVIEW

File: src/Lachain.Core/RPC/HTTP/Web3/BlockchainServiceWeb3.cs [hunk 1]
```diff
@@ -439,42 +456,30 @@ private JArray GetEventsByTransactionHash(string txHash)
...
+            if ((!(fromBlock is null) || !(toBlock is null)) && !(blockhash is null))
                 throw new Exception(
                     "If blockHash is present in in the filter criteria, then neither fromBlock nor toBlock are allowed.");
 
-            var start = (ulong)0;
+            ulong ?start;
+            start = _blockManager.GetHeight();
             if (!(fromBlock is null))
             {
-                if (((string)fromBlock!).StartsWith("0x"))
```
Comment (sgladkov) [inline src/Lachain.Core/RPC/HTTP/Web3/BlockchainServiceWeb3.cs:457]: If we use this functionality,  we should check if we can parse hex strings starting with "0x" and strings that are not started with special prefix. Add this logic to the GetBlockNumberByTag

Comment (tbssajal) [reply src/Lachain.Core/RPC/HTTP/Web3/BlockchainServiceWeb3.cs:457]: It is already implemented in Lachain.Utility.Utils.HexUtils.HexToUlong(this string buffer)

File: src/Lachain.Core/RPC/HTTP/Web3/BlockchainServiceWeb3.cs [hunk 2]
```diff
@@ -112,7 +112,13 @@ public class BlockchainServiceWeb3 : JsonRpcService
 
             if(fullTx) 
                 txArray = Web3DataFormatUtils.Web3BlockTransactionArray(txs, block!.Hash, block!.Header.Index);
-
+            else
```
Comment (sgladkov) [inline src/Lachain.Core/RPC/HTTP/Web3/BlockchainServiceWeb3.cs:115]: For the future: do not place different fixes in one pull request,  create other pull request if you need to fix something not related,  pull requests should have a narrow focus,  it really simplifies a work.

**Review by sgladkov — CHANGES_REQUESTED** [review]
Test_Web3Block is failed

File: src/Lachain.Core/RPC/HTTP/Web3/BlockchainServiceWeb3.cs (hunk 2 above)
Comment (tbssajal) [reply src/Lachain.Core/RPC/HTTP/Web3/BlockchainServiceWeb3.cs:115]: Okay, I will keep in mind.

**Review by sgladkov — APPROVED** [review]

**Pull request merged**